MSAL/test/unit/native_auth/cache/MSALNativeAuthCacheAccessorTest.swift

```diff
@@ -25,14 +25,24 @@
 import XCTest
 @testable import MSAL
 @_implementationOnly import MSAL_Private
+@_implementationOnly import MSAL_Unit_Test_Private
 
 final class MSALNativeAuthCacheAccessorTest: XCTestCase {
+#if os(iOS)
     private let tokenCache: MSIDDefaultTokenCacheAccessor = {
             let dataSource = MSIDKeychainTokenCache()
             return MSIDDefaultTokenCacheAccessor(dataSource: dataSource, otherCacheAccessors: [])
         }()
 
     private let accountMetadataCache: MSIDAccountMetadataCacheAccessor = MSIDAccountMetadataCacheAccessor(dataSource: MSIDKeychainTokenCache())
+#else
+    private let tokenCache: MSIDDefaultTokenCacheAccessor = {
```

Comment:
Do we need to explicitly check that the os is mac? This code will run also for visionOS (AFAIK msal objective-c is working on supporting this platform)